Dockerfile-pdfium.demo

```diff
@@ -0,0 +1,41 @@
+FROM trailofbits/polytracker
+MAINTAINER Carson Harmon <carson.harmon@trailofbits.com>
+
+WORKDIR /polytracker/the_klondike
+
+RUN apt-get install -y git pkg-config
+
+WORKDIR /
+
+RUN git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+ENV PATH=$PATH:/depot_tools/
+WORKDIR /polytracker/the_klondike/
+
+RUN gclient config --unmanaged https://pdfium.googlesource.com/pdfium.git
+RUN gclient sync
+
+RUN apt update
+RUN apt install software-properties-common -y
+RUN add-apt-repository "deb http://archive.canonical.com/ $(lsb_release -sc) partner"
+RUN apt update
+RUN apt install sudo -y
+RUN apt install vim -y
+#For some reason this isnt in our llvm-7 install?
+RUN apt install lld -y
+
+WORKDIR pdfium
+
+#Quick fix
+RUN patch -p1 < /polytracker/pdfium_patch
+
+RUN ln -fs /usr/share/zoneinfo/America/Chicago /etc/localtime
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
We probably only want to set `DEBIAN_FRONTEND` locally in the image-building commands that require that, and not set it persistently as an environment variable using `ENV`.